src/include/dav1d/dav1d.rs

```diff
@@ -209,10 +222,10 @@ impl From<Rav1dSettings> for Dav1dSettings {
             decode_frame_type,
         } = value;
         Self {
-            n_threads,
-            max_frame_delay,
+            n_threads: n_threads.get() as i32,
+            max_frame_delay: max_frame_delay.get() as i32,
```

Comment:
```suggestion
            n_threads: n_threads.get().into(),
            max_frame_delay: max_frame_delay.get().into(),
```

`u16` to `i32` should be lossless now, so `.into()` should work.